src/bot/lib/utils/currencies.js

```diff
@@ -0,0 +1,172 @@
+export const currencies = [
```

Comment:
Idk if this is possible but could we get this array from `Intl`? Like the supported list of currencies

Reply:
It's in the spec but not currently implemented

Referenced here
https://github.com/tc39/proposal-intl-enumeration and https://stackoverflow.com/a/58940564/1897184